buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -41,24 +41,6 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "Sync test native binaries",
```

Comment:
How does this relate to the intermedateAzureFeed issue?

Reply:
Reverting this change, thought this was syncing packages.